package.json

```diff
@@ -1,18 +1,31 @@
 {
   "name": "pariswebapp",
   "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-  },
+  "repository": "https://github.com/ziir/pariswebapp",
+  "homepage": "https://www.paris-web.fr/2018/ateliers/petits-trucs-pour-rendre-vos-applications-react-plus-performantes.php",
   "scripts": {
-    "precommit": "lint-staged"
+    "start": "NODE_ENV=production node index.js",
+    "dev": "nodemon index.js",
+    "precommit": "lint-staged",
+    "postinstall": "[ -z \"$VHOST\" ] && exit 0; ./build-all.sh && ./clean-all.sh",
+    "deploy": "ssh 840271@git.sd6.gpaas.net deploy default.git"
```

Comment:
I doubt this will be right for multiple people :-)

Reply:
Ahaha
It’s completely intentional, I want users to see how that can work :)
As this project serves a whole different purpose; this is just a bonus, regarding Gandi Simple Hosting Node.js it’s merely an example, not a guide or tutorial - yet ;)